packages/dds/tree/src/test/simple-tree/simpleTree.bench.ts

```diff
@@ -26,42 +27,355 @@ const nodesCountWide = [
 // number of nodes in test for deep trees
 const nodesCountDeep = [
 	[1, BenchmarkType.Measurement],
-	[10, BenchmarkType.Perspective],
-	[100, BenchmarkType.Measurement],
+	// [10, BenchmarkType.Perspective],
+	// [100, BenchmarkType.Measurement],
 ];
 
+describe.only("New benchmarks", () => {
```

Comment:
I added these so I could compare the scenarios against main. Might keep them in the end with better naming but not sure.